src/app/pages/MediaArticlePage/MediaArticlePage.tsx

```diff
@@ -131,6 +133,10 @@ const MediaArticlePage = ({ pageData }: { pageData: Article }) => {
     ...(isMap && { pageTitle: `${atiAnalytics.pageTitle} - ${brandName}` }),
   };
 
+  const isTransliterated =
+    ['serbian', 'zhongwen', 'uzbek'].includes(service) &&
+    pageType === 'article';
```

Comment:
Best to use the `ARTICLE_PAGE` const from `#app/routes/utils/pageTypes` here and in the ArticlePage component just for consistency.